snapcraft/tests/test_meta.py

```diff
@@ -217,251 +217,56 @@ def test_create_meta_with_icon_in_setup(self):
 
     def test_create_meta_with_app(self):
         os.makedirs(self.snap_dir)
-        open(os.path.join(self.snap_dir, 'app1.sh'), 'w').close()
+        open(os.path.join(self.snap_dir, 'app.sh'), 'w').close()
         self.config_data['apps'] = {
-            'app1': {'command': 'app1.sh'},
+            'app1': {'command': 'app.sh'},
+            'app2': {'command': 'app.sh', 'plugs': ['network']},
+            'app3': {'command': 'app.sh', 'plugs': ['network-server']}
```

Comment:
Yeah I `network-server` just to not forget about the concept of being able to do this, thanks @jdstrand .